data/secscan_model/secscan_v4_model.py

```diff
@@ -112,9 +115,17 @@ def __init__(self, app, instance_keys, storage):
             logger.warning(msg)
             raise InvalidConfigurationException(msg)
 
+        client = app.config.get("HTTPCLIENT")
+        cache_url = app.config.get("SECURITY_SCANNER_V4_CACHE", None)
+        if cache_url is not None:
+            c = Redis.from_url(cache_url)
+            backend = RedisCache(connection=c)
+            mixin = CacheMixin(backend=backend, cache_control=True)
```

Comment:
Is this caching everything including non-200s?

Reply:
`cache_control` uses headers to set expiration. Is that expected to be set from the responses from Clair?

Reply:
It's only caching 200 by default, and yes Clair sets `Cache-Control` headers.